code/iaas/model/src/main/java/io/cattle/platform/core/dao/impl/ClusterHostMapDaoImpl.java

```diff
@@ -41,4 +47,26 @@ public ClusterHostMapRecord getClusterHostMap(Host cluster, Host host) {
                 .fetchOneInto(ClusterHostMapRecord.class);
     }
 
+    @Override
+    public IpAddress getIpAddressForHost(Long hostId) {
+        return create()
+            .select(IP_ADDRESS.fields())
+                .from(IP_ADDRESS)
+                .join(HOST_IP_ADDRESS_MAP)
+                    .on(IP_ADDRESS.ID.eq(HOST_IP_ADDRESS_MAP.IP_ADDRESS_ID))
+                .where(HOST_IP_ADDRESS_MAP.HOST_ID.eq(hostId)
+                    .and(HOST_IP_ADDRESS_MAP.REMOVED.isNull())
+                    .and(IP_ADDRESS.REMOVED.isNull()))
+            .fetchOneInto(IpAddressRecord.class);
+    }
+
+    @Override
+    public Host findHostByName(String name) {
```

Comment:
I think I'll have to address this in a subsequent PR.  Maybe we can file an issue for it so we don't lose track of it.